.github/workflows/build-tests-mac.yml

```diff
@@ -20,11 +18,12 @@ jobs:
         projectPath:
           - test-project
         unityVersion:
-          - 2021.3.31f1
-          - 2022.3.11f1
-          - 2023.1.17f1
+          - 2021.3.32f1
+          - 2022.3.13f1
+          - 2023.1.19f1
```

Comment:
I think this is fine for now, but I don't think it should be a goal to have these updated to the latest all the time, as there's value in having reference versions. Especially for long running issues.